lib/spack/spack/build_environment.py

```diff
@@ -110,11 +110,13 @@
 
 
 class MakeExecutable(Executable):
-    """Special callable executable object for make so the user can
-       specify parallel or not on a per-invocation basis.  Using
-       'parallel' as a kwarg will override whatever the package's
-       global setting is, so you can either default to true or false
-       and override particular calls.
+    """Special callable executable object for make so the user can specify
+       parallelism options on a per-invocation basis.  Specifying
+       'parallel' to the call will override whatever the package's
+       global setting is, so you can either default to true or false and
+       override particular calls. Specifying 'jobs_env' to a particular
```

Comment:
IMO the following is clearer:

> If specified, `jobs_env` names the environment variable which should be used to control the level of parallelism

Reply:
Sounds reasonable, but I wanted to avoid giving the impression that the `-j{}` behavior would be turned off. How about:
>  If specified, jobs_env names an environment variable to be set to the number of parallel jobs used.
?

Reply:
That sounds good to me, although IMO if you want to make sure the user knows that `jobs_env` doesn't turn off the `-j` argument, that stating it explicitly would benefit either explanation.

Reply:
@scheibelp I think the latest commit resolves all your issues. Please let me know whether you concur.